gomock/matchers.go

```diff
@@ -139,3 +190,52 @@ func Not(x interface{}) Matcher {
 func AssignableToTypeOf(x interface{}) Matcher {
 	return assignableToTypeOfMatcher{reflect.TypeOf(x)}
 }
+
+// PredFunc is a Matcher that matches if the one argument function parameter
+// returns true for the given input
```

Comment:
nit: Can we make this commend doc a sentence with a period to match the other matchers?